controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -252,14 +255,83 @@ func (r *OCSInitializationReconciler) ensureRookCephOperatorConfigExists(initial
 			Namespace: initialData.Namespace,
 		},
 	}
-	err := r.Client.Create(r.ctx, rookCephOperatorConfig)
-	if err != nil && !errors.IsAlreadyExists(err) {
-		r.Log.Error(err, fmt.Sprintf("Failed to create %s configmap", util.RookCephOperatorConfigName))
+
+	csiPluginDefaults := defaults.DaemonPlacements["csi-plugin"]
+	csiProvisionerDefaults := defaults.DaemonPlacements["csi-provisioner"]
+
+	pluginTolerations := r.getCsiPluginTolerations()
+	provisionerTolerations := r.getCsiProvisionerTolerations()
```

Comment:
move this code inside CreateOrUpdate function

Reply:
`pluginTolerations` to `csiPluginTolerations` and `provisionerTolerations` to `csiProvisionerTolerations`

Reply:
Done